is_core/utils/__init__.py

```diff
@@ -77,132 +78,171 @@ def get_inline_views_opts_from_fieldsets(fieldsets):
     return inline_views
 
 
-def get_field_from_cls_or_inst_or_none(cls_or_inst, field_name):
-    from django.db.models.fields import FieldDoesNotExist
-
+def get_field_from_model_or_none(model, field_name):
     try:
-        return cls_or_inst._meta.get_field(field_name)
+        return model._meta.get_field(field_name)
     except (FieldDoesNotExist, AttributeError):
         return None
 
 
-def get_verbose_value(raw, field_or_method, obj, **kwargs):
+def _get_verbose_value(raw, field_or_method, obj, **kwargs):
```

Comment:
dal jsme to radši protectred. Je to spíš takové "nesahat" a moc vlastně ani nekoukat :D Vytáhnout tyto data z modelů, views, cores je fakt pěkný opruz